quartz_solar_forecast/forecast.py

```diff
@@ -30,6 +22,69 @@ def run_forecast(site: PVSite, ts: datetime | str = None, nwp_source: str = "ico
     pv_xr = make_pv_data(site=site, ts=ts)
 
     # load and run models
-    pred_df = forecast_v1(nwp_source, nwp_xr, pv_xr, ts)
+    pred_df = forecast_v1(nwp_source, nwp_xr, pv_xr, ts, model=model)
 
     return pred_df
+
+
+def predict_tryolabs(
+    site: PVSite, ts: datetime | str = None):
+    """Run the forecast with the tryolabs model"""
+
+    solar_power_predictor = TryolabsSolarPowerPredictor()
+
+    solar_power_predictor.load_model()
+    
+    if ts is None:
+        start_date = pd.Timestamp.now().strftime("%Y-%m-%d")
+        start_time = pd.Timestamp.now().floor("15min")
+    else:
+        start_date = pd.Timestamp(ts).strftime("%Y-%m-%d")
+        start_time = pd.Timestamp(ts).floor("15min")
+
+    end_time = start_time + pd.Timedelta(hours=48)
+
+    predictions = solar_power_predictor.predict_power_output(
+        latitude=site.latitude,
+        longitude=site.longitude,
+        start_date=start_date,
+        kwp=site.capacity_kwp,
+        orientation=site.orientation,
+        tilt=site.tilt,
+    )
+
+    if predictions is not None:
```

Comment:
why this if statement? 

Could you add a comment of why this is happen? Perhaps a loggin statement needs to happen if the predictions are None?